cpp/unreal_projects/SpearSim/Source/SpearSim.Target.cs

```diff
@@ -44,9 +42,11 @@ public SpearSimTarget(TargetInfo Target) : base(Target)
                     Path.Combine(plugin, "ThirdParty") + "=" +
                     Path.GetFullPath(Path.Combine(ProjectFile.Directory.FullName, "..", "..", "..", "third_party"));
             }
-            
-        } else {
-            throw new Exception("[SPEAR | SpearSim.Target.cs] Target.Platform == " + Target.Platform);            
+        // we can't throw an exception here because when we invoke GenerateProjectFiles.bat/GenerateProjectFiles.sh script, Unreal Build Tool
```

Comment:
Add whitespace before this comment, and move it outside of the the else-if statement
```

        // We can't throw an exception here, because when we invoke GenerateProjectFiles.bat or GenerateProjectFiles.sh (packaged with
        // the Unreal Engine), instances of this class get created with Target.Platform set to many different platforms (e.g., iOS, tvOS,
        // Win32, etc).
        // } else {
        //    throw new Exception("[SPEAR | SpearSim.Target.cs] Target.Platform == " + Target.Platform);            
        // }
```